docusaurus/docs/reactnative/guides/video_attachment.mdx

```diff
@@ -0,0 +1,141 @@
+---
+id: video-attachment
+sidebar_position: 6
+title: Video Attachment
+---
+
+We would be building a custom video component in the application using some of the available open-source packages and using the `FileAttachment` customization prop which the `Channel` component allows us to use.
+
+Uploading a video attachment is possible using the file upload in the attachment picker.
+
+To add the video attachment and view the file we follow these steps:
+
+## Installing the packages
+
+We use two packages for rendering the video and creating the thumbnail of the video.
+For rendering the video in the message list we recommend using the [react-native-video](https://www.npmjs.com/package/react-native-video) package.
+To create the thumbnail of the video attachment we recommend using the [react-native-create-thumbnail](https://github.com/souvik-ghosh/react-native-create-thumbnail) package.
+
+Note: [react-native-video](https://www.npmjs.com/package/react-native-video) and [react-native-create-thumbnail](https://github.com/souvik-ghosh/react-native-create-thumbnail) are external package.
+
+After installing it do not forget to link the package to your project. Follow the guide in the documentation for Android and iOS installation for `react-native-video` - [here](https://github.com/react-native-video/react-native-video#installation) and for `react-native-create-thumbnail` - [here](https://github.com/souvik-ghosh/react-native-create-thumbnail#getting-started).
+
```

Comment:
Linking isn't required from react native 0.60 and upwards, and we require newer versions of react native as a peer dependency, so I'm surprised that users need to manually link the packages. Could you help me understand what makes it important to do it manually? Is there something about how our SDK is set up that makes it so?

Reply:
So, for android, the linking didn't work automatically for the package `react-native-video` in my case while I was setting it up. Therefore, I mentioned it to the users. It's their choice if they want to refer it or not but it's always good to have a direct link to those guides imo.

Reply:
I agree that it's good to link it, it just sounded like users absolutely have to. Maybe it can be phrased along the lines of "If you encounter any problems installing the dependencies, please refer to their installation and troubleshooting docs here" or something like that? I didn't encounter any problems while installing it, and read through their docs to figure out what I was missing/what error I was meant to look for 😅